packages/components/addon/components/hds/application-state/footer.hbs

```diff
@@ -0,0 +1,7 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div class={{this.classNames}} ...attributes>
+  {{yield (hash Link::Standalone=(component "hds/link/standalone"))}}
```

Comment:
For reviewers: I don't super love `Link::Standalone` but it was already used this way in the Alert component. I think `Link` would be nicer and more consistent with the other hash yield names we use. WDYT?

Reply:
In the `Alert` actions, where there are different possible type of actions, we used the full name to make it clear which one it's used. In this case only one type is possible, so could be considered redundant. 
At the same time, can give an indication to the consumers of what arguments to use (eg. `@icon` is required on the standalone link, not in the inline link).
That said, no strong preference: I see pros and cons in both solutions.